### DISCUSSION TITLE
The Stop attribute in the CreateRequest

### DISCUSSION THREAD

**Discussion by Immerseit** [betalgo/openai#42] [Q&A] [answered]
Hi

How do I insert a List of stop sequences? According to the python, curl etc versions, I should insert an array. But in this library, there is a nullable string.

Thanks,

**Comment by kayhantolga** [comment]
The library wasn't supporting string or string array types. I just created a Pull Request for you. https://github.com/betalgo/openai/pull/43

I will give it a bit of time for code review before release.

**Comment by kayhantolga** [comment, marked as answer]
Version 6.6.1 released.  
You can use StopAsList parameter if you want to pass string list

**Comment by Immerseit** [comment]
Great! Thanks. Do you support the finetunes-features? Different question. I
know.

Den tis 27 dec. 2022 14:16Tolga Kayhan ***@***.***> skrev:

**Reply by kayhantolga** [reply]
Yeap, sample is available in playground project.

**Reply by Immerseit** [reply]
I have tried it but eventually I do not understand the API or there is a bug. The SetDefaultEngineId is not allowed to be null in finetuning. But if I set it, I got an error that I can't mix Engines and Models. But there appear to be no way to get around it.

**Reply by kayhantolga** [reply]
I will have a look at this. They changed the way of use models in API a few months ago. There might be a bug related to these changes.
